bundles/multitenant-managed-api-service/bundle.Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Execute this docker file via make bundle/build command with VERSION env
+
+FROM scratch
+
+ARG version
+ARG manifest_path=bundles/multitenant-managed-api-service/${version}/manifests
+ARG metadata_path=bundles/multitenant-managed-api-service/${version}/metadata
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=multitenant-managed-api-service
```

Comment:
This will need to change to `managed-api-service` otherwise the package is still `multitenant-managed-api-service` and upgrade fails.

Reply:
Sorry not sure how I missed this but it's been fixed in the latest push.